### PR TITLE
Feature/issue expo 39 upgrade

### PR DESCRIPTION
**Pull request by ian-lin-PG**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "ts-jest": "^26.4.1",
     "tslint": "^6.1.3",
     "tslint-immutable": "^6.0.1",
-    "typescript": "~3.9.2"
+    "typescript": "^4.0.3"
```
Comment (rasmus-storjohann-PG) [inline package.json:29]: This is exciting, spent a little bit of time looking at what's new and have some ideas for improvements in how we use the language.

File: tsconfig.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
         "module": "es2015",
         "moduleResolution": "node",
         "jsx": "react-native",
+        "incremental": true,
```
Comment (rasmus-storjohann-PG) [inline tsconfig.json:7]: I saw my build time go from 14s to 5s with this change. Let's try it out and see if it has any downsides.

File: src/components/loading_screen/loading_service_list_component.tsx [hunk 3]
```diff
@@ -54,6 +54,6 @@ const LoadingSpinner = (): JSX.Element => (
         alignItems: 'center',
         justifyContent: 'center',
       }}>
-          <ActivityIndicator size='large' />
+          <ActivityIndicator size='large' color={colors.teal} />
```
Comment (tomy-pg) [inline src/components/loading_screen/loading_service_list_component.tsx:57]: lets take some time auditing the app to see if any more issues have risen with the `react-native` and `native-base` upgrade.

**Comment by tomy-pg** [comment]
Good team effort on this PR.

**Pull request merged**